doc/redirects.json

```diff
@@ -3679,21 +3679,6 @@
   "rust-package-built-with-meson": [
     "index.html#rust-package-built-with-meson"
   ],
-  "compiling-rust-crates-using-nix-instead-of-cargo": [
```

Comment:
Please don't simply delete old locations, but redirect them to release notes. That way whoever has a bookmark will instantly find the reason why the thing they were using is gone.